### DISCUSSION TITLE
Email alerts

### DISCUSSION THREAD

**Discussion by 12nick12** [owncast/owncast#1614] [Ideas]
I would love to have an email alert set up when I go live. For example someone subscribes and when I go live on the site an email is sent out, via SMTP/API (AWS SES).

**Comment by gabek** [comment]
I posted an idea recently about having built-in ways to send notifications, but I'm not sold that all the additional dependencies would be worth it. That's over at https://github.com/owncast/owncast/issues/1609. I'm afraid it would bloat up the project a lot for something not that many people would use. But It'd be cool.

However, you can do this now yourself by using the webhook functionality.  I have an example of how you might do that with https://glitch.com/~owncast-example-follow, but it's just an example. It uses Mailgun as the SMTP server to send emails but you could rework it using AWS if you wanted.

**Reply by 12nick12** [reply]
Oh awesome, thank you. I didn't even think about the webhook.
